tests/test_default_qubit.py

```diff
@@ -366,8 +366,11 @@ def test_apply_errors(self):
             p = [np.array([1, 0, 1, 1, 1])/np.sqrt(3)]
             self.dev.apply('QubitStateVector', wires=[0, 1], par=[p])
 
-        with self.assertRaisesRegex(ValueError, "BasisState parameter must be an array of len\(wires\) many 0/1 integers\."):
-            self.dev.apply('BasisState', wires=[0, 1], par=[np.array([0, 1, 4.2])])
+        with self.assertRaisesRegex(ValueError, "BasisState parameter must be an array of 0 or 1 integers of length at most {}.".format(2)):
```

Comment:
Is this a typo? Why use `format`?

Reply:
No, this is not a typo. I kept the format for two reasons: 1) laziness, I just copy and pasted this line from `template.py` 2) By keeping the check similar to the code that is checked, small changes in the former also result in minimal changes in the later. If you insist, I can change it :-)

Reply:
lol I insist :)
It just looks weird when you see it in isolation

Reply:
fixed :)